### DISCUSSION TITLE
How to do conditional input rendering?

### DISCUSSION THREAD

**Discussion by matheusliraofficial** [orchidsoftware/platform#1531] [Q&A]
Hello guys!

How can I render one Input in form based on checkbox or select input value?

**Comment by bilogic** [comment]
I'm facing the same question and I think this could answer it
https://orchid.software/en/docs/listener/

**Reply by brianmutiso** [reply]
I have the same issue, this fixes it but there is a problem when you are using a model on form, when your input names are in the format model.field it doesn't work, I don't know why

eg:    protected $targets = [
        'merchant_product.product_id',
    ];

doesn't work but when I change to just product_id it works, however i can't use my model

**Reply by brianmutiso** [reply]
**Update:** Only fields of type Input::make() are not working, Select and Relation is ok 

error in input fields:

 vendor.js?id=d88d437…:2 Uncaught (in promise) Error: Request failed with status code 500
htmlspecialchars expects parameter 1 to be string, array given
